src/health_misinfo_shared/vertex.py

```diff
@@ -108,4 +111,8 @@ def generate_training_set(folders: list[str], label: str):
         "prostate_cancer_nat_rem",
         "weight_loss_nat_rem",
     ]
-    generate_training_set(_folders, label="natural_remedies_x5")
+    # generate_training_set(_folders, label="natural_remedies_x5")
+    all_videos_in_folder = youtube_api.load_texts(_folders[0])
+    for video in all_videos_in_folder:
+        process_video(video)
+        wibble
```

Comment:
There’s a stray wibble here

Reply:
Good spot, but it just means the little ad hoc test script will terminate a bit early.